File: app/src/main/java/com/gomez/herlin/logindemo/retrofit/RetrofitApiService.java
package com.gomez.herlin.logindemo.retrofit;

import com.gomez.herlin.logindemo.dto.DonutsDto;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface RetrofitApiService {

    @GET("donuts")
    Call<List<DonutsDto>> getDonuts();
}

File: app/src/main/java/com/gomez/herlin/logindemo/retrofit/RetrofitApiService.kt
package com.gomez.herlin.logindemo.retrofit

import com.gomez.herlin.logindemo.dto.DonutsDto
import retrofit2.Call
import retrofit2.http.GET

interface RetrofitApiService {
    @get:GET("donuts")
    val donuts: Call<List<DonutsDto?>?>?
}
